MAINTAIN.md

```diff
@@ -79,6 +79,8 @@ When a (non-experimental) feature is slated to be removed it should:
       `v0.10.0-dev-1957+gd676746c33` then use `0.12`.
     - For Vimscript features, use `v:lua.vim.deprecate()`. Use the same version
       as described for Lua features.
+    - `vim.deprecate(…, 'x.y.z')` where major version `x` is greater than the
+      current Nvim major version, is always treated as _soft_ deprecation.
```

Comment:
cc @dundargoc this is the only policy that makes sense when `vim.deprecate(...,'1.x')` is called

Reply:
I agree, at least now when we're zerover. It's possible some adjustments might need to be made after we're no longer zerover, but there's no reason to think on that now.